documentation/MeetingMinutes/CAMARA Device Location - 2024-03-26.md

```diff
@@ -0,0 +1,127 @@
+  - Waiting for Commonalities outcomes
+  - As we have several issues aroung Subscriptions and the progress on commonalities are slow we need to speed up the resolution. One proposal should be to have a cross meeting with Device Status WG as these 2 projets rely on subscriptions.
+
+* [Geofencing API - Defining a standard behavior for first event](https://github.com/camaraproject/DeviceLocation/issues/124)
+  - Moved to Commonalities [Issue 140](https://github.com/camaraproject/Commonalities/issues/140)
+  - No comments in the issue in the last 2 weeks
+  - Low participation in commonalities is blocking the issue for device location
+  - Ludovic proposes to have a cross meeting with Device Location and Device Status to tackle the issue
+
+
+* [Define guidelines for geofencing implementation](https://github.com/camaraproject/DeviceLocation/issues/133)
+  - No new comments. 
+  - [Action] We need more comments from the team
+  - Connected with [Issue #85](https://github.com/camaraproject/DeviceLocation/issues/85). A document with implementation guideline should cover this also.
+
+* [Geofencing API - Add "format: uri" to notificationUrl](https://github.com/camaraproject/DeviceLocation/issues/118)
+  - Moved to [commonalities](https://github.com/camaraproject/Commonalities/issues/93)
+  - No comments in the last 2 weeks
+  - Wait for outcome
+
+### Long term discussions
+
+#### Administrative Code Area
+
+* New [Issue #83](https://github.com/camaraproject/DeviceLocation/issues/83), with formal requirements from GSMA Product track
+  - [Document uploaded by TEF](https://github.com/camaraproject/DeviceLocation/files/12856149/AdminCode.Proposal.-.Draft_20230926.docx) with a proposal. 
+
+  This item should be reopen after MWC.
+
+  - We should discussed approach on this for next release. Should we incorporate this to current operations, should we split this functionality to new endpoints or even separate APIs?
+    - Need to be discussed
+      - Ludovic preference is for a separate API as for example in France this is not a request.
+      - For Javier this is not a top priority for Telefonica as of now.
+    - More generally probably we need to have feedback loop with GSMA team about API work (and review the priority as focus could change)
+      - Done during last TSC - see here: https://wiki.camaraproject.org/display/CAM/2024-03-21+TSC+Minutes
+
+#### Behaviour when requesting too high precision
+
+* [Issue #85](https://github.com/camaraproject/DeviceLocation/issues/85)
+  - As requested by Joachim, issue will remain open until TEF uploads the document with more detail or the information is added to an API_documentation file to keep in the repo for future references.
+    - Joachim: Should be also aligned/synchronysed with Geofencing
+
+## AOB
+
+<p>
+
+- Next call would be on **April 9th, 2024**
+- **Not present in calendar** : Ludovic will check with José about resending a serie - we have probaly to move to LF zoom.

```

Comment:
Yes, we'll have to move to Confluence and Zoom. I'll prepare next agenda in the Wiki and resend the email with the new link